indico/modules/events/registration/blueprint.py

```diff
@@ -47,6 +54,25 @@
                  'close_regform', RHRegistrationFormClose, methods=('POST',))
 _bp.add_url_rule('/manage/registration/<int:reg_form_id>/schedule',
                  'schedule_regform', RHRegistrationFormSchedule, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/modify/', 'modify_regform', RHModifyRegForm)
+
+# Sections
```

Comment:
I'd go for something like this to make it clear

    # Regform edition: sections